### ISSUE TITLE
Long description criticisms

### ISSUE THREAD

**Issue by dotsdl** [datreant/scipy-2016#2]
From @orbeckst on the long description, for reference:

It's ok, but not great.
1. I suggest a re-read of Schimel, Science Writing (especially ch 5, Opening)
2. The first paragraph takes too long to develop the problem; a second sentence starting with "furthermore" is a complete waste of prime attention real estate.
3. datreant is buried in the morass of the middle.
4. Too wordy.
5. I would leave the name's etymology to the talk.
6. More about the problems that datreant solves and how it fills a niche.
7. Why is MDSynthesis not mentioned? You already have an interesting application (that you presented on last year) and it's not even mentioned?? (see #1)

**Comment by kain88-de** [comment]
I have another point out the waste of _prime attention real estate_. Don't make say why something else is bad rather say why your solution is to good. I would delete the whole database part and use that to mention MDSynthesis and use more space about the extensibility of datreant.

**Comment by dotsdl** [comment]
Addressed these comments in 6b705db4a404ae05a455c1979577f3e358fd9e70. I'll leave this open, though, for more discussion on the long description until we're happy with it.

**Comment by dotsdl** [comment]
If we're satisfied with this, I'll close. Any more changes needed for long description? Beginning to sculpt abstract from it (#4).

@sseyler? I think you said you had some thoughts.

**Comment by orbeckst** [comment]
Sounds good to me.

I don't know if links show up as links everywhere; if in doubt I would spell out all urls.

On 28 Mar, 2016, at 11:57, David Dotson wrote:

> If we're satisfied with this, I'll close. Any more changes needed for long description? Beginning to sculpt abstract from it (#4).
> 
> �
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub

## 

Oliver Beckstein \* orbeckst@gmx.net
skype: orbeckst  \* orbeckst@gmail.com

**Comment by dotsdl** [comment]
Will do. "Final" version up now (thanks @sseyler for further refinements). Preparing abstract from it.

**Comment by dotsdl** [comment]
Any more changes feel free to bring up here or directly in line comments.

**Comment by dotsdl** [comment]
Okay, so I made some additions/modifications to the long description in light of in-person discussion with @orbeckst. I think it's more convincing now, which is important especially since this is what gets graded by the talk selection committee.
